### PR TITLE
[fix] class calls

### PR DESCRIPTION
**Pull request by yenda**
fixes #julien

## Sumary

This concerns:
- touchable-opacity
- animated-view
- scroll-view

## Tip for rebasing:
search in project for `scroll-view)` `animated-view)` and `touchable-opacity)`

## Testing

Tested on Android:
- scroll views in chat
- animated view when chat is loading
- touchable opacity on ropsten -> chat -> stickers -> "get stickers"

status: ready

### CODE REVIEW

File: components/src/status_im/ui/components/react.cljs [hunk 1]
```diff
@@ -94,9 +94,13 @@
 
 (def pan-responder (lazy-get-react-property "PanResponder"))
 (def animated (lazy-get-react-property "Animated"))
-(defn animated-view []
+
+(defn animated-view-class []
   (reagent/adapt-react-class (.-View (animated))))
 
+(defn animated-view [props content]
+  [animated-view-class props content])
```
Comment (yenda) [inline components/src/status_im/ui/components/react.cljs]: @rasom is it correct here I wasn't sure about this one it's a bit different than the other

**Review by mandrigin — APPROVED** [review]

**Review by rasom — APPROVED** [review]

**Review by vitvly — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by churik** [comment]
@yenda can you please resolve conflicts?

**Comment by yenda** [comment]
@churik it's done

**Comment by asemiankevich** [comment]
@yenda 

chat bar is missing on **iOS** if i want to start a conversation from user profile. Steps:

1. on device A : tap plus button -> scan QR code (or Start new chat)
2. on device B: profile -> share my contact code
3. as device A scan the code shown on device B -> tap send message

image

I checked and it works in  nightly. Would you mind taking a look?

**Comment by yenda** [comment]
@asemiankevich I don't understand are you sure you tested the right build? I just tried latest one on iphone 6 and it see the input field

**Comment by asemiankevich** [comment]
> @asemiankevich I don't understand are you sure you tested the right build? I just tried latest one on iphone 6 and it see the input field

let me recheck now, i tested on iPhone 7

**Comment by yenda** [comment]
@asemiankevich ok thanks, that'd be very confusing if it doesn't show because this is suppose to be equivalent to what we had before

**Comment by asemiankevich** [comment]
> @asemiankevich ok thanks, that'd be very confusing if it doesn't show because this is suppose to be equivalent to what we had before

@yenda can't reproduce now

**Comment by yenda** [comment]
@asemiankevich cool so yeah it was just that you tested the previous build that also failed the end2end tests

is it good to merge then?

**Comment by asemiankevich** [comment]
@yenda i would test it a bit more if possible. won't take long.

**Comment by yenda** [comment]
@asemiankevich I protest! all you are going to find is bugs from other PRs and blame it on mine :D

**Pull request merged**